opdracht-1.js

```diff
@@ -4,26 +4,57 @@
 // Gebruik een array-methode om een array te maken met alle tv-type namen.
 // Log de uitkomst in de console.
 
-
+const tvType = inventory.map((inventory ) => {
+    return inventory.type;
+});
+console.log(tvType);
 
 //----------------------------------------------------------------------------------------------------------------------
 
 // Opdracht 1b:
 // Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn.
 // Log de uitkomst in de console.
 
-
+const outOfStock = inventory.filter((inventory ) => {
+    const originalStock = inventory.originalStock
+    let soldOut = inventory.sold;
+    if (originalStock === soldOut){
+        return inventory;
+    }
+});
```

Comment:
De filter-functie verwacht een boolean waarde, en je geeft de inventory terug. Daarnaast kun je de functie nog wat inkorten
```suggestion
const outOfStock = inventory.filter(inventory => originalStock === soldOut);
```